qcodes/math/field_vector.py

```diff
@@ -1,3 +1,8 @@
+"""
+A convenient class to keep track of vectors representing physical fields.
```

Comment:
Sorry to be such a nitpicker, but the module docstring should document the module, which is not a class. So smth like "A helper module containing a class to keep track of vectors in different coordinate systems" would be more on point.